### PR TITLE
Fix ingress creation parameter inline documentation

### PR DESCRIPTION
**Pull request by biglittlebigben**

### CODE REVIEW

File: packages/livekit-server-sdk/src/IngressClient.ts [hunk 1]
```diff
@@ -22,11 +22,11 @@ export interface CreateIngressOptions {
...
   /**
-   * name of the room to send media to. optional
+   * name of the room to send media to.
    */
   roomName?: string;
   /**
-   * unique identity of the participant. optional
+   * unique identity of the participant.
    */
   participantIdentity?: string;
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/IngressClient.ts:31]: since they are not optional, we shouldn't make them `?`. and should move these to the top?

Comment (biglittlebigben) [reply packages/livekit-server-sdk/src/IngressClient.ts:31]: Would that potentially break any existing code using the SDK?

Comment (davidzhao) [reply packages/livekit-server-sdk/src/IngressClient.ts:31]: if this is required, then existing code would either have to set it, or it wouldn't be successful either way, right?

File: packages/livekit-server-sdk/src/IngressClient.ts [hunk 2]
```diff
@@ -67,11 +67,11 @@ export interface UpdateIngressOptions {
    */
   name: string;
   /**
-   * name of the room to send media to. optional
+   * name of the room to send media to.
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/IngressClient.ts:70]: this is optional right? on updates.

Comment (biglittlebigben) [reply packages/livekit-server-sdk/src/IngressClient.ts:70]: Everything is optional in the dictionary in the sense that parameters left out will not be updated. It's odd to single out these 2 parameters. We could flag all parameters as optional, but this seems redundant on a parameter called "UpdateIngressOptions".

Comment (davidzhao) [reply packages/livekit-server-sdk/src/IngressClient.ts:70]: got it.. makes sense.

File: packages/livekit-server-sdk/src/IngressClient.ts [hunk 3]
```diff
@@ -22,11 +22,11 @@ export interface CreateIngressOptions {
...
   /**
-   * name of the room to send media to. optional
+   * name of the room to send media to. required
    */
   roomName?: string;
   /**
-   * unique identity of the participant. optional
+   * unique identity of the participant. required
    */
   participantIdentity?: string;
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/IngressClient.ts:31]:
```suggestion
   */
  participantIdentity: string;
```

File: packages/livekit-server-sdk/src/IngressClient.ts [hunk 4]
```diff
@@ -146,17 +146,27 @@ export class IngressClient extends ServiceBase {
...
-      video = opts.video;
-      participantMetadata = opts.participantMetadata;
+    if (opts == null) {
+      throw new Error('options dictionary is required');
+    }
+
+    name = opts.name || '';
+    roomName = opts.roomName;
+    participantName = opts.participantName || '';
+    participantIdentity = opts.participantIdentity || '';
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/IngressClient.ts:156]:
```suggestion
    participantIdentity = opts.participantIdentity;
```

File: packages/livekit-server-sdk/src/IngressClient.ts [hunk 5]
```diff
@@ -146,17 +146,27 @@ export class IngressClient extends ServiceBase {
...
-      enableTranscoding = opts.enableTranscoding;
-      url = opts.url || '';
-      audio = opts.audio;
-      video = opts.video;
-      participantMetadata = opts.participantMetadata;
+    if (opts == null) {
+      throw new Error('options dictionary is required');
+    }
+
+    name = opts.name || '';
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/IngressClient.ts:153]: need consts in front of these

**Pull request merged**